setup.py

```diff
@@ -17,6 +17,7 @@
     zip_safe=False,
     install_requires=[
         "microcosm>=2.12.0",
+        "microcosm-eventsource>=1.0.0",
```

Comment:
Adding our event sourcing library for tracking the `order` state machine with the `order_event` model!